environments/sandbox/sandbox.yml

```diff
@@ -80,6 +80,17 @@ A:
     record:
       - "20.26.30.175"
 
+  - name: "labs-goldenpath-naraynm"
+    ttl: 300
+    record:
+      - "51.145.114.159"
+
+  - name: "labs-goldenpath-naraynm-temp"
```

Comment:
maybe we call this `labs-goldenpath-naraynm-lb` and the other `labs-goldenpath-naraynm-fw` not massively particular though, just a thought